src/style.css

```diff
@@ -1 +1,94 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
```

Comment:
The same issue here with the `font-family` property in the `@font-face` rule. Please, correct it.